### PR TITLE
Odo operators services

### PR DESCRIPTION
**Pull request by yhontyk**
- branches 4.4 and 4.5
- reviewed and approved by the SMEs/QE

### CODE REVIEW

File: cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-the-operators.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[id=creating-instances-of-services-managed-by-the-operators]
+= Creating instances of services managed by the Operators
```
Comment (bmcelvee) [inline cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-the-operators.adoc:2]: As written, the title is a bit awkward. Maybe something along the lines of, "Creating instances of services managed by Operators" could work?

File: cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-the-operators.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+:context: creating-instances-of-services-managed-by-the-operators
+toc::[]
+
+:FeatureName: Creating instances of services managed by the Operators  in {odo-title}
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Operators are a method of packaging, deploying, and managing Kubernetes services. With odo, you can create instances of services from the Custom Resource Definitions (CRDs) provided by the Operators. You can then use these instances in your projects and link them to your components..
+
+To create services from an Operator, you must ensure that the Operator has valid values defined in its `metadata` to start the requested service. odo uses the `metadata.annotations.alm-examples` YAMl file of an Operator to start
+the service. If this YAML has placeholder values or sample values, a service cannot start. You can modify the YAML file and start the service with the modified values. See xref:Creating services from a YAML file
```
Comment (bmcelvee) [inline cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-the-operators.adoc:14]: "xref:Creating services" is an incomplete link, and the sentence is incomplete.

File: cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-the-operators.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+.Prerequisites
+    • `oc` is installed
```
Comment (bmcelvee) [inline cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-the-operators.adoc:17]: Formatting for bullets is: 
"* Install the `oc` CLI."

s/• `oc` is installed/* Install the `oc` CLI and log in. 

For the following, maybe make it a sub-bullet (as follows), and for the "see Adding Operators to the cluster" part, you can link to another assembly. 
** Note that the configuration of the cluster determines the services available to you. To access Operator services, a cluster administrator must install the respective Operator on the cluster first. To learn more, see Adding Operators to the cluster. 

s/• Latest version of `odo` is installed/* Install the `odo` CLI. 
s/• Experimental mode is enabled./* Enable experimental mode. 
** To enable experimental mode in `odo` preferences, run:
<add code block, and format the code example: ---->
`odo preference set Experimental true` or use the environment variable `odo config set --env ODO_EXPERIMENTAL=true`
You can also use the `ODO_EXPERIMENTAL=true` prefix when running commands, for example `ODO_EXPERIMENTAL=true odo catalog list services`

File: modules/developer-cli-odo-creating-a-service-from-an-operator.adoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-the-operators.adoc
+
+[id="creating-a-service-from-an-operator_{context}"]
+
+= Creating a service from an operator
```
Comment (bmcelvee) [inline modules/developer-cli-odo-creating-a-service-from-an-operator.adoc:7]: s/operator/Operator

File: modules/developer-cli-odo-creating-services-from-yaml-files.adoc [hunk 5]
```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-the-operators.adoc
+
+[id="creating-services-from-yaml-files_{context}"]
+
+= Creating services from YAML files
+
+If the YAML definition of the service or Custom Resource (CR) has invalid or placeholder data , you can use the `--dry-run` flag to get the YAML definition, specify the correct values and start the service using the corrected YAML definition.
```
Comment (bmcelvee) [inline modules/developer-cli-odo-creating-services-from-yaml-files.adoc:9]: s/data ,/data, 
s/correct values and/correct values, and
s/service (Custom Resource or CR)/service or CR

File: modules/developer-cli-odo-creating-services-from-yaml-files.adoc [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+odo provides the feature to print the YAML definition of the service (Custom
+Resource or CR) provided by the Operator before starting a service. 
+
+. To display the YAML of the service, run:
++
+----
+$ odo service create <operator-name> --crd <cr-name> --dry-run
+----
++
+For example, to print YAML definition of `EtcdCluster` provided by the `etcdoperator.v0.9.4` Operator:
```
Comment (bmcelvee) [inline modules/developer-cli-odo-creating-services-from-yaml-files.adoc:20]: s/Operator/Operator, run

File: modules/developer-cli-odo-listing-available-services-from-the-operators-installed-on-the-cluster.adoc [hunk 7]
```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-the-operators.adoc
+
+[id="listing-available-services-from-the-operators-installed-on-the-cluster_{context}"]
+
+= Listing available services from the Operators installed on the cluster
+
```
Comment (bmcelvee) [inline modules/developer-cli-odo-listing-available-services-from-the-operators-installed-on-the-cluster.adoc:8]: Add an introductory sentence or statement

**Comment by bmcelvee** [comment]
I left a couple of nits and suggestions. Do you have an issue or bug that you could link to this PR? Thanks! :)

**Comment by boczkowska** [comment]
@bmcelvee thank you so much for the review! I've addressed all you comments, PTAL when you have time.
Regarding an issue - we track our issues in Jira (https://issues.redhat.com/browse/RHDEVDOCS-1610) since we plan our work separately form OpenShift docs.

**Pull request merged**